jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml

```diff
@@ -312,6 +312,11 @@
             NOTE: tempest does not support testing RADOS Gateway, so failures on object_storage
             tempest tests are expected when ses_rgw_enabled is set to true.
 
+      - bool:
+          name: openid_connect_enabled
+          default: '{openid_connect_enabled|true}'
```

Comment:
Do we want it to be enabled by default? What impact does doing so have on an cloud deployment?

Reply:
We want to make sure it does not break anything by enabling it by default. This would provide *some* CI coverage. Later on, we'll need to see if we can find a creative way to run functional tests on it.

Reply:
Ok so the point is that enabling this _**shouldn't**_ break things for a normal deployment, and thus we want it enabled to catch if something we submit changes that state of affairs.